packages/open-banking/visa/src/app/page.tsx

```diff
@@ -0,0 +1,4 @@
+import { redirect } from "next/navigation";
+export default function Home() {
+    redirect("/create-customer"); // redirect to /create-external-party
```

Comment:
```suggestion
    redirect("/create-customer"); // redirect to /create-customer
```

Reply:
Resolved by this [commit](https://github.com/Dwolla/integration-examples/pull/32/commits/01edca6d98079d67f2c7c1f5af271996c50312e4)! Thanks!